index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="container">
+        <header class="nav-header">
+            <a href="index.html">
+                <img src="./media-files/Logo.png" alt="My Logo">
+            </a>
+            <nav>
+                <img src="./media-files/Enabled.png" alt="Menu">
+            </nav>
+        </header>
```

Comment:
- Great thinking removing the unnecessary icons on the Figma UI but kindly check the comparison attached below and find how the header section is not centrally aligned on the y (main) axis, Champ.
- [ ] I see you followed the requirement of `flexbox`as he display tool well but if you include an àlign-items: center` and remove the margin I can almost spot at the top, I think you will have a perfectly aligned header.